hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/MetadataScanResult.java

```diff
@@ -63,7 +63,7 @@ public boolean isDeleted() {
   }
 
   @Override
-  public boolean isHealthy() {
+  public boolean hasErrors() {
     return errors.isEmpty();
```

Comment:
With the method renamed its meaning is changed, so the return value should be inverted.  Previously `true` meant "OK" and `false` meant "problem".  Now it should be the other way around.

```suggestion
  public boolean hasErrors() {
    return !errors.isEmpty();
```

Callers also need to be changed to account for that.

Reply:
@adoroszlai  Agree!  made changes to address this